Tasks/VsTest/task.json

```diff
@@ -209,7 +222,7 @@
             "groupName": "reportingOptions"
         }
     ],
-    "instanceNameFormat": "Test Assemblies $(testAssembly)",
+    "instanceNameFormat": "Test Assemblies",
```

Comment:
Input value removed from title due to an issue with newline character in task title. We need to update this update this, once build team fixes this issue.